src/index.ts

```diff
@@ -257,7 +319,12 @@ class Logging {
    * region_tag:logging_create_sink
    * Another example:
    */
-  createSink(name: string, config, callback) {
+  createSink(name: string, config: CreateSinkRequest): Promise<[Sink, LogSink]>;
+  createSink(
+      name: string, config: CreateSinkRequest,
+      callback: CreateSinkCallback): void;
+  async createSink(name: string, config: CreateSinkRequest):
+      Promise<[Sink, LogSink]> {
     // jscs:enable maximumLineLength
```

Comment:
Not your code, but can you yank this line?

Reply:
Sure thing